ci-operator/step-registry/dpf/hypervisor/setup-cluster-access/dpf-hypervisor-setup-cluster-access-commands.sh

```
@@ -0,0 +1,36 @@
+
+REMOTE_HOST="${REMOTE_HOST:-10.6.135.45}"
+CLUSTER_NAME=$(cat "${CLUSTER_PROFILE_DIR}/cluster-name")
+PROXY_PORT=8213
+
+cat /var/run/dpf-ci/private-key | base64 -d > /tmp/id_rsa
+echo "" >> /tmp/id_rsa
+chmod 600 /tmp/id_rsa
+SSH_OPTS="-i /tmp/id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR -o ConnectTimeout=30 -o ServerAliveInterval=10 -o ServerAliveCountMax=3 -o BatchMode=yes"
+
+echo "Testing SSH to ${REMOTE_HOST}..."
+ssh ${SSH_OPTS} root@${REMOTE_HOST} echo 'SSH OK'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)